src/components/playback/playqueuemanager.js

```diff
@@ -176,21 +208,39 @@ define([], function () {
 
     PlayQueueManager.prototype.reset = function () {
 
+        this._sortedPlaylist = [];
         this._playlist = [];
         this._currentPlaylistItemId = null;
         this._repeatMode = 'RepeatNone';
+        this._shuffleMode = 'Sorted';
     };
 
     PlayQueueManager.prototype.setRepeatMode = function (value) {
-
-        this._repeatMode = value;
+        if (value === 'RepeatOne' || value === 'RepeatAll' || value === 'RepeatNone') {
+            this._repeatMode = value;
+        } else {
+            throw new TypeError('invalid value provided for setRepeatMode');
```

Comment:
@MrTimscampi Yeah, with the current code is impossible as well. I thought about adding this here basically in case someone wants to touch something in the remote players and don't have a clue why shuffling doesn't work :)